### ISSUE TITLE
Emacs port feedback

### ISSUE THREAD

**Issue by wasamasa** [whatyouhide/vim-gotham#5]
[It happened](https://github.com/wasamasa/gotham-theme).

I've tried matching the colors from your screenshot as closely as possible. It would be very useful if you could provide me extra screenshots and point out obvious mistakes. For instance I'm less sure about comments having almost the same color as the Ruby keywords. Once all of them are fixed, I'll proceed theming more packages and will add the mappings for the terminal version.

**Comment by wasamasa** [comment]
I've started working on my port again (after like, six months?) and finished up most of the requested packages, however I've noticed that there are a few I cannot theme as well as I'd like to, namely the ERC and Circe packages.

Both are IRC clients for Emacs that support mIRC-style color escapes and can turn these into 16 colors, not unlike what a terminal emulator supports. While there is a good amount of shades for gray and blueish hues, there are a lot less for the remaining colours. I'm not sure whether the same issue applies to Vim (since Neovim got support for fast terminal emulation and could possibly affected by theming, too), but would like an official statement on this to be able to mark these packages as finished.
